Misc/NEWS.d/next/Core and Builtins/2023-05-31-13-54-41.gh-issue-105140.JmPdPc.rst

```diff
@@ -0,0 +1 @@
+Remove unused, untested and undocumented arg of ``_PyErr_ChainStackItem``.
```

Comment:
I don't think that end users care about changes in the internal C API. You can remove this NEWS entry.